src/api/app/views/webui/users/tokens/edit.html.haml

```diff
@@ -0,0 +1,25 @@
+.card
+  .card-body
+    %h3.mb-3 Edit Token
+
+    .row
+      .col-12.col-md-10.col-lg-8
+        -# "as: :token" to have a consistent name across all various token classes
+        = form_for(@token, as: :token, url: token_path, method: :put, local: true) do |f|
+          .form-row#edit-token
+            .col-12.col-md-10.col-lg-9
+              .form-group
+                = f.label(:id, 'Id:')
+                = @token.id
+              .form-group
+                = f.label(:type, 'Operation:')
+                = @token.class.token_name.capitalize
+              .form-group
+                = f.label(:scm_token) do
+                  SCM token:
+                  %abbr.text-danger{ title: 'required' } *
+                .input-group
+                  = f.password_field(:scm_token, size: 80, class: 'form-control', placeholder: 'Please enter your new SCM token', required: true)
```

Comment:
True, I won't use a password field.

Reply:
With a password field, we cannot potentially leak the SCM token even when a workflow token already has a SCM token. It will simply display the placeholder in this case. With a normal input field, the SCM token is always displayed. I prefer to never show the SCM token since this is how SCMs handle their tokens.

I'll change the "Create Token" form to use a password field.

Reply:
> I'll change the "Create Token" form to use a password field.

Done, please have another look

Reply:
Fine! :+1: